frontend/nuxt.config.ts

```diff
@@ -106,80 +51,89 @@ const openverseLocales = [
-          cert: fs.readFileSync(path.resolve(__dirname, "localhost+1.pem")),
-        }
-      : undefined,
-  },
-  router: {
-    middleware: "feature-flags",
-  },
-  plugins: [
-    "~/plugins/ua-parse.ts",
-    "~/plugins/focus-visible.client.ts",
-    "~/plugins/api-token.server.ts",
-    "~/plugins/polyfills.client.ts",
-    "~/plugins/sentry.ts",
-    "~/plugins/analytics.ts",
-    "~/plugins/errors.ts",
-    "~/plugins/init-stores.ts",
-  ],
-  css: ["~/assets/fonts.css", "~/styles/tailwind.css", "~/styles/accent.css"],
-  head,
-  env, // TODO: Replace with `publicRuntimeConfig`
-  privateRuntimeConfig: {
-    apiClientId: process.env.API_CLIENT_ID,
-    apiClientSecret: process.env.API_CLIENT_SECRET,
+  serverDir: "server/",
+  devServer: {
+    port: 8443,
+    host: "0.0.0.0",
+  },
+  imports: {
+    autoImport: false,
+  },
+  css: ["~/assets/fonts.css", "~/styles/accent.css"],
+  runtimeConfig: {
+    apiClientId: "",
+    apiClientSecret: "",
+    public: {
+      deploymentEnv: process.env.DEPLOYMENT_ENV ?? "local",
+      // Can be overwritten by NUXT_PUBLIC_API_URL env variable
+      apiUrl,
+      providerUpdateFrequency: 3600000,
+      savedSearchCount: 4,
+      sentry: {
+        dsn: isProdNotPlaywright
+          ? "https://b6466b74788a4a2f8a7912eea912beb7@o787041.ingest.sentry.io/5799642"
+          : "",
+        environment: isProd ? "production" : "local",
+        release: "",
+      },
+      isPlaywright,
+    },
   },
   dev: !isProd,
-  buildModules: [
-    "@nuxt/typescript-build",
-    "@nuxtjs/composition-api/module",
-    "@nuxtjs/svg-sprite",
-    "@pinia/nuxt",
-  ],
+  /**
+   * Disable debug mode to prevent excessive timing logs.
+   */
+  debug: false,
+  experimental: {
+    /**
+     * Improve router performance, see https://nuxt.com/blog/v3-10#%EF%B8%8F-build-time-route-metadata
+     */
+    scanPageMeta: true,
+  },
   modules: [
-    "portal-vue/nuxt",
+    "@pinia/nuxt",
     "@nuxtjs/i18n",
-    "@nuxtjs/proxy",
-    "@nuxtjs/redirect-module",
-    "@nuxtjs/sentry",
-    "cookie-universal-nuxt",
-    "vue-plausible",
-    "~/modules/prometheus.ts",
-    // Sitemap must be last to ensure that even routes created by other modules are added
+    "@nuxtjs/tailwindcss",
+    "@nuxtjs/plausible",
+    "@nuxt/test-utils/module",
     "@nuxtjs/sitemap",
   ],
-  serverMiddleware: [
-    { path: "/healthcheck", handler: "~/server-middleware/healthcheck.js" },
-    { path: "/robots.txt", handler: "~/server-middleware/robots.js" },
-  ],
-  svgSprite: {
-    input: "~/assets/svg/raw",
-    output: "~/assets/svg/sprite",
+  routeRules: {
+    "/photos/**": { redirect: { to: "/image/**", statusCode: 301 } },
+    "/meta-search": { redirect: { to: "/about", statusCode: 301 } },
+    "/external-sources": { redirect: { to: "/about", statusCode: 301 } },
+  },
+  tailwindcss: {
+    cssPath: "~/styles/tailwind.css",
   },
   i18n: {
     baseUrl: "https://openverse.org",
```

Comment:
I know this isn't changed in the diff, but why doesn't this match the actual deployment URL? Shouldn't this be `staging.openverse.org` for staging, or does it not matter?

Reply:
The docs say that this should not be used if we set the VueRouter's base URL (at least in Nuxt 3):

> The fallback base URL to use as a prefix for alternate URLs in hreflang tags. By default VueRouter's base URL will be used and only if that is not available, fallback URL will be used.

However, in staging, all `hreflang` alternate links do use `https://openverse.org`. I'll check what happens with Nuxt 3 if we remove this setting